lib/dal/src/context.rs

```diff
@@ -524,10 +524,12 @@ impl DalContext {
         let events_actor = self.events_actor();
 
         let rebase_batch_address = slow_rt::spawn(async move {
```

Comment:
Not important for this PR, but I'm reminded I wanted to move the `slow_rt::spawn` calls to use the `self.compute_executor.spawn()` calls instead.

Reply:
Some follow on work, no doubt (that would need independent testing confirmation)